src/components/styles/Layout.tsx

```diff
@@ -64,10 +64,12 @@ export const TableControlOuterRow = styled(TableControlInnerRow)`
 
 export const PageButtonRow = styled.div`
   display: flex;
-  button {
-    margin-left: 24px;
+  justify-content: flex-end;
+  > button,
+  > a {
+    margin-left: 16px;
+    white-space: nowrap;
   }
```

Comment:
This can be more generic and not rely on the implementation details of the child components.
```suggestion
  > * {
    margin-right: 16px;
  }
  > *:last-child {
    margin-right: 0;
  }
```